fiftyone/utils/annotations.py

```diff
@@ -955,11 +955,12 @@ def _format_attributes(backend, attributes):
         if not attr:
             attr = backend.recommend_attr_tool(name, None)
 
-        attr_type = attr.get("type", None)
-        values = attr.get("values", None)
-        default = attr.get("default", None)
-        mutable = attr.get("mutable", None)
-        read_only = attr.get("read_only", None)
+        _attr = attr.copy()
+        attr_type = _attr.get("type", None)
+        values = _attr.get("values", None)
```

Comment:
@ehofesmann FYI I believe this slightly changes how `_attr` is constructed.

For example:
- Previously if `attr["mutable"] == None`, then `_attr` would **not** contain a `"mutable"` key, since it is only added if it is non-`None`
- Now as implemented, `_attr` would contain a `"mutable"` key with value `None`

I'm not sure if that change in behavior breaks anything, but you could maintain the current behavior if you used `pop()` instead of `get()`:

```py
attr_type = _attr.pop("type", None)
values = _attr.pop("values", None)
default = _attr.pop("default", None)
mutable = _attr.pop("mutable", None)
read_only = _attr.pop("read_only", None)
```